hydra-node/src/Hydra/API/Server.hs

```diff
@@ -370,9 +387,11 @@ handleDraftCommitUtxo directChain tracer body reqMethod reqPaths respond = do
               CannotCommitReferenceScript -> return400 e
               CommittedTooMuchADAForMainnet _ _ -> return400 e
               UnsupportedLegacyOutput _ -> return400 e
+              walletUtxoErr@SpendingNodeUtxoForbidden -> return400 walletUtxoErr
```

Comment:
ErrCoverFee are not covered and considered 500 status code, is that a problem?

Reply:
Unless we run into lookup problems before, maybe the `ErrUnknownInput` could be raised due to a user picking a `TxIn` which is unknown.

So yes, I would say that error case would also be a 400